nginx.conf

```diff
@@ -36,7 +36,7 @@ http {
         listen       8080 default_server;
         listen       [::]:8080 default_server;
         server_name  _;
-        root         /srv/repo;
+        root         /srv/tree/repo;
```

Comment:
The old location looks correct to me reading https://github.com/openshift/os/commit/ffbc3285b757ebc10e71b7dced04172cbd964ab1#diff-3254677a7917c6c01f55212f86c57fbfR14

Reply:
I didn't test this locally actually but...see the problem, we need to keep both container builds in sync.  It's tempting to neuter the base one until such time as we switch to a pure-container solution.